### PR TITLE
Reorder download & open app content for clarity

### PR DESCRIPTION
**Pull request by jaffrepaul**
Reorder some content to give priority to the most-taken actions from common users & encourage action.

### CODE REVIEW

File: content/guides/getting-started/installing-cypress.md [hunk 1]
```diff
@@ -12,57 +12,8 @@ title: Installing Cypress
...
-  ui:
-    image: cypress/base:latest
-    # if targeting a specific node version, use e.g.
-    # image: cypress/base:14
-```
-
-`cypress/base` is a drop-in replacement for
-[base docker node images](https://hub.docker.com/_/node/).
+First, make sure you have all the [system requirements](#System-requirements),
+and then...
```
Comment (elylucas) [inline content/guides/getting-started/installing-cypress.md:16]:
```suggestion
First, make sure you have all the [system requirements](#System-requirements).
```

**Review by elylucas — APPROVED** [review]

File: content/guides/getting-started/installing-cypress.md (hunk 1 above)
Comment (elylucas) [reply content/guides/getting-started/installing-cypress.md:16]: This make it back in someone or did you want to keep it?

File: content/guides/getting-started/opening-the-app.md [hunk 2]
```diff
@@ -15,37 +15,41 @@ title: Opening the App
 
 ## `cypress open`
 
-If you used `npm` to install, Cypress has now been installed to your
+<Alert type="info">
```
Comment (elylucas) [inline content/guides/getting-started/opening-the-app.md:18]: not even sure we need this alert section and its content

**Review by elylucas — APPROVED** [review]

File: content/guides/getting-started/installing-cypress.md (hunk 1 above)
Comment (jaffrepaul) [reply content/guides/getting-started/installing-cypress.md:16]: Oops l undid some changes from earlier. This is what happens when you poke at PRs between stuff at a conf 😀 I’ll pull it in again.

Comment (jaffrepaul) [reply content/guides/getting-started/installing-cypress.md:16]: Oops l undid some changes from earlier. This is what happens when you poke at PRs between stuff at a conf 😀 I’ll pull it in again.

File: content/guides/getting-started/opening-the-app.md (hunk 2 above)
Comment (jaffrepaul) [reply content/guides/getting-started/opening-the-app.md:18]: Yeah it seems a little excessive. I only kept it and wrapped the alert to tie it to the “Long version” using the full path we discuss below. 

We may not really need that either. I believe it’s only for users not using npm or a script, etc. not clear on how many people would do that. 

TMI? We could just pull these bits.

**Comment by elylucas** [comment]
Ya I think pull it

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request merged**